src/transform/src/normalize_lets.rs

```diff
@@ -798,9 +798,6 @@ mod inlining {
                             &expr
                         };
                         match stripped_value {
-                            // TODO: One could imagine CSEing multiple occurrences of a global Get
-                            // to make us read from Persist only once.
-                            // See <https://github.com/MaterializeInc/database-issues/issues/6363>
```

Comment:
Was this comment ever right? I thought we dedup reads from persist at the source importing moment. Was it removed because it wasn't right? Or some other reason?

Reply:
It was never right!

> I thought we dedup reads from persist at the source importing moment.

That's correct!